core/Objects/CmfPackage.cs

```diff
@@ -647,9 +648,44 @@ public static CmfPackage Load(IFileInfo file, bool setDefaultValues = false, IFi
             cmfPackage.Location = PackageLocation.Local;
             cmfPackage.fileSystem = fileSystem;
 
+            cmfPackage.RelatedPackages?.Load(cmfPackage);
+
             return cmfPackage;
         }
 
+        public void Load(bool setDefaultValues = false)
+        {
+            if (!FileInfo.Exists)
+            {
+                throw new CliException(string.Format(CoreMessages.NotFound, FileInfo.FullName));
+            }
+
+            string fileContent = FileInfo.ReadToString();
+            JsonConvert.PopulateObject(fileContent, this);
+            IsToSetDefaultValues = setDefaultValues;
+            Location = PackageLocation.Local;
+
+            RelatedPackages?.Load(this);
+
+        }
+
+        public void Pick()

```

Comment:
should be Peek unless I'm misunderstanding the purpose of the method

Reply:
you are correct